magma/fromverilog.py

```diff
@@ -107,7 +107,7 @@ def ParseVerilogModule(node, type_map):
 
     return node.name, args
 
-def FromVerilog(source, func, type_map, module=None):
+def FromVerilog(source, func, type_map, target_modules=None):
```

Comment:
Is it worth having both a single module argument and a list of modules argument? It would probably be convenient, but I'm not sure if there's an elegant way to do this without having to introduce some complexity in the code (handling the various cases). Maybe we can have a separate function that defines a single module from verilog (and then wraps the module in a list of one element and passes it to this function).

Reply:
Seems like reducing the complexity here and forcing the client to pass in a list and take the 0-th element of the list is preferable, esp. since it is likely not the most common case. This change also moves in the direction of thinking about bigger/general/more complex design scenarios where people might implement several modules in the same file etc.

Reply:
Makes sense, I agree.